### ISSUE TITLE
Node 12 compilation problems

### ISSUE THREAD

**Issue by ZJONSSON** [astro/node-expat#195]
@astro Looks like major changes in v8 are causing problems with node-expat under node 12.   Did a quick stab fixing the obvious things but this seems to be relatively complicated and possibly interrelated with NaN
For reference see https://github.com/nodejs/node/issues/23122 and  https://github.com/bcoin-org/bcrypto/issues/7

```
./node-expat.cc: At global scope:
../node-expat.cc:11:26: error: ‘Handle’ has not been declared
   static void Initialize(Handle<Object> target)
                          ^~~~~~
../node-expat.cc:11:32: error: expected ‘,’ or ‘...’ before ‘<’ token
   static void Initialize(Handle<Object> target)
                                ^
../node-expat.cc:481:23: error: ‘Handle’ has not been declared
   void Emit(int argc, Handle<Value> argv[])
                       ^~~~~~
../node-expat.cc:481:29: error: expected ‘,’ or ‘...’ before ‘<’ token
   void Emit(int argc, Handle<Value> argv[])
                             ^
../node-expat.cc: In static member function ‘static void Parser::Initialize(int)’:
../node-expat.cc:29:5: error: ‘target’ was not declared in this scope
     target->Set(Nan::New("Parser").ToLocalChecked(), t->GetFunction());
     ^~~~~~
../node-expat.cc:29:5: note: suggested alternative: ‘Parse’
     target->Set(Nan::New("Parser").ToLocalChecked(), t->GetFunction());
     ^~~~~~
     Parse
../node-expat.cc:29:69: error: no matching function for call to ‘v8::FunctionTemplate::GetFunction()’
     target->Set(Nan::New("Parser").ToLocalChecked(), t->GetFunction());
....
```

(for reference I have no problems doing the same install under node 11)

**Comment by Y--** [comment]
Hi all!

Thanks for this great package!
I've made a PR for Node 12 here https://github.com/astro/node-expat/pull/196

Let me know if I missed anything.

Thanks!

**Comment by Y--** [comment]
This should be fixed now. I've released version 2.3.18 that supports Node 12.
Let me know if I've missed anything.
